Makefile

```diff
@@ -1,110 +1,85 @@
-SHA=$(shell git rev-parse --short HEAD)
-export DIRTY=$(shell if `git diff-index --quiet HEAD --`; then echo false; else echo true;  fi)
-export BASE_BINARY_NAME=terraform-provider-snowflake
 export GO111MODULE=on
 export TF_ACC_TERRAFORM_VERSION=1.4.1
 export SKIP_EXTERNAL_TABLE_TESTS=true
 export SKIP_SCIM_INTEGRATION_TESTS=true
 
-go_test ?= -
-ifeq (, $(shell which gotest))
-	go_test=go test
-else
-	go_test=gotest
-endif
+BASE_BINARY_NAME=terraform-provider-snowflake
+TERRAFORM_PLUGINS_DIR=$(HOME)/.terraform.d/plugins
+TERRAFORM_PLUGIN_LOCAL_INSTALL=$(TERRAFORM_PLUGINS_DIR)/$(BASE_BINARY_NAME)
+COVERAGE_REPORT_FILE=coverage.txt
+COVERAGE_FLAGS=-coverprofile=$(COVERAGE_REPORT_FILE) -covermode=atomic
 
-all: test docs install
-.PHONY: all
+help: ## display help for this makefile
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
noice [noice](https://media.tenor.com/2Px7xeNIBswAAAAd/mr-bean-magic.gif)

Reply:
that's not mine, the file is just reorganised